dev-lang/python/metadata.xml

```diff
@@ -26,6 +26,11 @@
 			by running Python's test suite and collecting statistics
 			based on its performance. This will take longer to build.
 		</flag>
+		<flag name="tail-interp">
```

Comment:
Why not follow upstream and name it `tail-call-interp`?

Reply:
I wasn't sure if it was too long ;)